Tools/Scripts/libraries/webkitbugspy/webkitbugspy/bugzilla.py

```diff
@@ -679,6 +679,7 @@ def cc_radar(self, issue, block=False, timeout=None, radar=None):
                 )
                 if response == 'No':
                     raise ValueError('Radar is tracking a different bug')
+                user_to_cc = True  # Ensure that the user override is respected even if 'InRadar' is applied
```

Comment:
Maybe this is unfamiliarity with this code, but I'm very confused as to how we have a variable called `user_to_cc` which has anything to do `InRadar`? Unless the user is the importer user?

Reply:
> Unless the user is the importer user?

Indeed!